fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/agg/GroupConcat.java

```diff
@@ -100,6 +102,13 @@ public GroupConcat(Expression arg0, Expression arg1, OrderExpression... orders)
     public GroupConcat(boolean distinct, boolean alwaysNullable, int nonOrderArguments, List<Expression> args) {
         super("group_concat", distinct, alwaysNullable, args);
         this.nonOrderArguments = nonOrderArguments;
+        checkArguments();
+    }
+
+    @Override
+    public boolean nullable() {
+        return children().stream()
+                .anyMatch(expression -> !(expression instanceof OrderExpression) && expression.nullable());
```

Comment:
this is not right, we need to process scalar agg(no group by), so the class implement `NullableAggregateFunction`, when `alwaysNullable` set to `true`, `nullable()` need to return `true`.